api/pkg/pagination/pagination.go

```diff
@@ -7,9 +7,9 @@ import (
 
 // Paging can be used to capture paging information in API responses.
 type Paging struct {
-	Page  int32
-	Pages int32
-	Total int32
+	Page  int32 `json:"page,omitempty"`
+	Pages int32 `json:"pages,omitempty"`
+	Total int32 `json:"total,omitempty"`
```

Comment:
i didn't get the part on snake case, is there `paging_page` used or generated anywhere?

Reply:
My bad. That is meant to be "lower case" and not "snake case". I fixed the description now.